devices.js

```diff
@@ -2706,6 +2706,23 @@ const devices = [
         },
     },
 
+    // Gear Axis
+    {
+        zigbeeModel: ['Gear'],
+        model: 'Gear',
```

Comment:
Is there some model number written on the device? This is usually what we will put here.

Reply:
Just check it should be GR-ZB01-W, let me update it